test/Microsoft.AspNet.Server.KestrelTests/EngineTests.cs

```diff
@@ -116,6 +116,22 @@ public void ConnectionCanReadAndWrite(TestServiceContext testContext)
 
             Console.WriteLine("Started");
             var socket = new Socket(AddressFamily.InterNetwork, SocketType.Stream, ProtocolType.Tcp);
+            if (PlatformApis.IsWindows)
+            {
+                const int SIO_LOOPBACK_FAST_PATH = (-1744830448);
+                byte[] optionInValue = BitConverter.GetBytes(1);
+                try
+                {
+                    socket.IOControl(SIO_LOOPBACK_FAST_PATH, optionInValue, null);
+                }
+                catch (Exception e)
```

Comment:
OK, if CoreCLR doesn't support it properly yet it should be fine to catch `Exception` since these are just tests.  I was just being pedantic :smile: